### DISCUSSION TITLE
Slickgrid is collapsing back to 50% when I drag one area over the other to 100%.

### DISCUSSION THREAD

**Discussion by nelefelt** [6pac/SlickGrid#975] [Q&A]
Hi, I have a problem with my slickgrid that consists of 2 areas. If I drag the upper or lower part over the other area so that a part covers 100% of the surface, the areas collapse back again to 50% / 50% when I release the bar. In my local storage which saves the sizes, it still says that one occupies 0% size and the other 100%, but when I check in dev tools, the flex property is set to 50% on each.

The strange thing is that it works every other time, where the first time it collapses to 50%/50%, but every other time after that, it sticks as desired in the 100% position.

Has anyone dealt with the same issue or knows what possible direction i should go for solving this?

**Comment by 6pac** [comment]
Hi @Drobba, 
We'll need more information about what the config is. Not sure what you mean by two areas or how you're getting them.
Ideally, provide an example that runs in the repo examples folder. You can attach the example file to the reply.
Then give instructions to repro the problem.
